gogoproto/gogo.proto

```diff
@@ -103,5 +107,11 @@ extend google.protobuf.FieldOptions {
   optional string jsontag = 65005;
   optional string moretags = 65006;
   optional string casttype = 65007;
+	optional FieldValidator validate = 65008;
```

Comment:
I think this is the only one that is needed, at least for version one, but maybe I am missing something?